app/lib/map_graph.py

```diff
@@ -0,0 +1,33 @@
+def map_graph(graph):
```

Comment:
If this function is only needed for limiting the graph  you can put it in the `limit_graph.py` 
Please also add description of this function.

Reply:
I figured since map_graph maps the indices of connected nodes, it can be applied to other use cases when we need to further process the graph.